src/app/home/home.component.spec.ts

```diff
@@ -22,4 +22,62 @@ describe('HomeComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it("Should use the 'Grin' emote when category is not yet set", () => {
+    component.ngOnInit();
+    const homeElement: HTMLImageElement = fixture.nativeElement;
+    const emote = homeElement.querySelector('img')!;
+    expect(emote.src).toBe("https://cdn.naomi.lgbt/emotes/NaomiGrin.png")
+  });
```

Comment:
```suggestion
  it('should default to the question view', () => {
    expect(component.view).toBe('question');
  });
```

`ngOnInit()` runs when the component loads. The tests load the component in a headless browser.

For this test, we can assert that the category is `question` on load, because we'll test the view properly in a subsequent suite.

Reply:
It's actually a pretty neat idea. Though your codebase uses 'category' not 'view'. Thanks for the lovely suggestion.